src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2323,6 +2323,10 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             var currentHomeSpace = Workspaces.OfType<HomeWorkspaceModel>().FirstOrDefault();
             currentHomeSpace.UndefineCBNFunctionDefinitions();
 
+            // This is to handle the case of opening a JSON file that does not have a version string
+            EngineController.WorkspaceVersion = dynamoPreferences.Version ==
```

Comment:
did you see any of these files?

Reply:
No, I haven't seen any. This comment has been copied from deleted lines of code below.